.github/workflows/ci.yml

```diff
@@ -18,8 +18,6 @@ jobs:
     strategy:
       matrix:
         go-version: [
-          "1.20",
-          "1.21",
           "1.22",
```

Comment:
Tried to add 1.23 as well, but got error
```
Setup go version spec 1.23
Attempting to download 1.23...
matching 1.23...
Not found in manifest.  Falling back to download directly from Go
```

I think idea here was "lets test in OADP version (1.20) for compatibility, NAC version (1.21) for sanity and next (1.22) to see if we can update when needed"

do we want still test if updates can be done or just testing on current go version is enough?

Reply:
I think that current go version is sufficient at this point. The app is containerized, so this should not be a problem?

Reply:
yeah

Velero and OADP also tests only current Go version, so I believe is ok to only test on Go version we are using (only enforcing should be that Velero, OADP and NAC use same Go version)